docs/scos/dev/guidelines/nfr-guidelines.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Project Development Non-Functional Requirement guidelines
+description: @TODO
+last_updated: @TODO
+template: concept-topic-template
+related:
+- title: Operatable feature guidelines
+  link: docs/scos/dev/guidelines/operatable-feature-guidelines.html
+- title: Process documentation guidelines
+  link: docs/scos/dev/guidelines/process-documentation-guidelines.html
+- title: Monitorable process guidelines
+  link: docs/scos/dev/guidelines/monitorable-process-guidelines.html
+---
+
+
+## Introduction
```

Comment:
In general it can eb nice to add specific examples to each and every case + describe what we provide OOTB

Reply:
I'd love to provide what we have OOTB, but unfortunately currently we don't have this available. As soon as we'll have meaningful NFRs to share, we will, but priorities make it hard to work on them.